scripts/common/host-packages.sh

```diff
@@ -95,6 +95,22 @@ function dpkg_install_host_packages() {
     _dpkg_install_host_packages "$dir" "$dir_prefix" "${packages[@]}"
 }
 
+function _dpkg_apt_get_status_and_maybe_fix_broken_pkgs() {
+    logStep "Checking package status"
+    if  sudo apt-get check status ; then
+        logSuccess "Status checked successfully. No broken packages were found."
+        return
+    fi
+
+    logWarn "Attempting to correct broken packages by running apt-get install -f -y"
+    sudo apt-get install -f -y
```

Comment:
_A bit of a nit pick here_

Perhaps writing the full command as `sudo apt-get install --fix-broken --yes` here would make things explicit for future readers

Reply:
`-f` at times can be misunderstood as `--force`

Reply:
There is also `--fix-missing` option in the mix that makes things even more confusing :)

Reply:
Hun, if we run --help I think we can check that is fix-broken and yes. 
I think we can move forward with right. That does not seems a blocker for we get this one merged.

Reply:
Done.

Reply:
For sure you can do that. It would be helpful if I'm reading code I don't have to search for apt-get man pages or google around to see what this options mean. Just making a readers life easier. Not a blocker anyway.